files/architecture/security.tex

```diff
@@ -0,0 +1,27 @@
+\chapter{Security}
+This chapter describes the considerations and decisions taken to secure the back-end application.
+
+\section{Authorization}
+To make sure that malicious users can't abuse the back-end application, the decision to secure it through authorization was made. There was 2 options for how this could be handled. 
+
+\subsection{The common approach}
+This approach is to add a token of sorts to the header of the http request. The downside of this option when working with GraphQL is that you can not add a http header to your request in graphiql when you query from the browser, unless a browser extension for that specific thing is used. It would also not be very transparent or obvious when browsing graphiql that a authentication token is required for certain endpoints.
```

Comment:
"http" -> "HTTP"
gls on "http", "GraphQL", "graphqil"
Add a reference to the browser extension.
"a authentication token" -> "an authentication token"
"endpoints" could maybe be "resources" instead, if we are very strict on this term.